### PR TITLE
Support fusion options for benchmark.py

### PR DESCRIPTION
**Pull request by zhangyaobit**
**Description**: Describe your changes.

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: onnxruntime/python/tools/transformers/benchmark.py [hunk 1]
```diff
@@ -99,18 +100,20 @@ def run_onnxruntime(use_gpu, provider, model_names, model_class, config_modifier
...
+            args.model_type = MODELS[model_name][3]
+            fusion_options = FusionOptions.parse(args)
 
             if 'pt' in model_source:
                 with torch.no_grad():
                     onnx_model_file, is_valid_onnx_model, vocab_size, max_sequence_length = export_onnx_model_from_pt(
                         model_name, MODELS[model_name][1], MODELS[model_name][2], MODELS[model_name][3], model_class,
                         config_modifier, cache_dir, onnx_dir, input_names, use_gpu, precision, optimizer_info,
-                        validate_onnx, use_raw_attention_mask, overwrite, model_fusion_statistics)
+                        validate_onnx, use_raw_attention_mask, overwrite, model_fusion_statistics, fusion_options)
```
Comment (wangyems) [inline onnxruntime/python/tools/transformers/benchmark.py:111]: might log some warning as fusion_options will only take effect when optimizer_info is set to by_script.
https://github.com/microsoft/onnxruntime/blob/e03b799b957a1828b2c470213ca3feac7edee78a/onnxruntime/python/tools/transformers/benchmark_helper.py#L34

File: onnxruntime/python/tools/transformers/benchmark.py [hunk 2]
```diff
@@ -517,6 +520,8 @@ def parse_arguments():
                         default=None,
                         help="Manually set the model's layer number")
 
+    FusionOptions.add_arguments(parser)
```
Comment (ytaous) [inline onnxruntime/python/tools/transformers/benchmark.py:523]: >parser [](http://example.com/codeflow?start=32&length=6)

can we append one example of fusion options here?
e.g. --disable_embed_layer_norm

**Comment by ytaous** [comment]
>         Run ONNXRuntime and TorchScript on CPU for all models with quantization:

can we append one example of fusion options here?
e.g. --disable_embed_layer_norm

---
Refers to: onnxruntime/python/tools/transformers/benchmark.py:35 in 0c79f92. [](commit_id = 0c79f92557163b4c6ddf363351ac09150ffa0a3e, deletion_comment = False)

File: onnxruntime/python/tools/transformers/benchmark.py (hunk 1 above)
Comment (zhangyaobit) [reply onnxruntime/python/tools/transformers/benchmark.py:116]: Added a warning.

**Comment by zhangyaobit** [comment]
> > ```
> >     Run ONNXRuntime and TorchScript on CPU for all models with quantization:
> > ```
> 
> can we append one example of fusion options here? e.g. --disable_embed_layer_norm
> 
> Refers to: onnxruntime/python/tools/transformers/benchmark.py:35 in [0c79f92](https://github.com/microsoft/onnxruntime/commit/0c79f92557163b4c6ddf363351ac09150ffa0a3e). [](commit_id = [0c79f92](https://github.com/microsoft/onnxruntime/commit/0c79f92557163b4c6ddf363351ac09150ffa0a3e), deletion_comment = False)

Yes, added an example.

**Review by wangyems — APPROVED** [review]
lgtm

**Review by ytaous — APPROVED** [review]

**Pull request merged**